config.go

```diff
@@ -36,6 +36,7 @@ func parseFlags() {
 	flag.String("localstatedir", "", "Directory to store local state information. (default: workdir)")
 	flag.String("rundir", "", "Directory to store runtime data. (default: workdir/run)")
 	flag.String("logdir", "", "Directory to store logs. (default: workdir/log)")
+	flag.String("logfile", "-", "Log file name. (default: -)")
```

Comment:
I meant to point to update the help text only. The rest is fine. YUpdate the help text to `Log file name. (logs to stderr by default)`.